defaults/main.yml

```diff
@@ -50,6 +48,11 @@ __network_packages_default_team: ["{%
       if __network_team_connections_defined
       %}NetworkManager-team{% endif %}"]
 
+# if wireless or team connection is specified, NetworkManager should be
+# allowed to restart by default
+network_allow_restart: "{{ true if __network_wireless_connections_defined
```

Comment:
```yaml
network_allow_restart: "{{ __network_wireless_connections_defined or
  __network_team_connections_defined }}"
```
Ansible style says not to use boolean `true` and `false` if you can just use the result of a boolean context evaluation